packages/orchestrator-ui-components/src/components/WFOSubscription/WFOSubscriptionGeneral.tsx

```diff
@@ -14,13 +22,35 @@ export const WFOSubscriptionGeneral = ({
     subscriptionDetail,
 }: WFOSubscriptionGeneralProps) => {
     const t = useTranslations('subscriptions.detail');
+    const { theme } = useOrchestratorTheme();
+
+    const getInSyncIcon = (value: boolean) => {
```

Comment:
We have the same logic/icon in `packages/orchestrator-ui-components/src/components/WFOSubscriptionsList/WFOSubscriptionsList.tsx` . You could extract a reusable component from that

Reply:
Created a separate WFOInsyncIcon component